Italian translation 1.4

Dasc3er opened the pull request:
Pull request for the Italian translation (#236).

Dasc3er committed:
Italian translation 1.4

amencarini commented on version/1/4/it/code_of_conduct.md:
```diff
@@ -0,0 +1,47 @@
+
+## Le Nostre Responsabilità
+
+I mantenitori del progetto sono responsabili nel chiarire gli standard di comportamento accettabile e sono supposti di prendere azioni correttive appropriate e giuste in risposta a qualsiasi caso di comportamento inaccettabile.
```
I **responsabili** del progetto sono **tenuti a** chiarire gli standard di comportamento accettabile **e a prendere** azioni correttive appropriate e giuste in risposta a qualsiasi caso di comportamento inaccettabile.

amencarini commented on version/1/4/it/code_of_conduct.md:
```diff
@@ -0,0 +1,47 @@
+
+I manutentori del progetto che non seguono o applicano il Codice di Condotta in buona fede possono affrontare ripercussioni temporanee o permanenti in base alla deliberazione di altri membri della direzione del progetto.
+
+## Riferimenti
```
**Attribuzione**

amencarini commented on version/1/4/it/code_of_conduct.md:
```diff
@@ -0,0 +1,47 @@
+Casi di comportamento ingiurioso, molesto o altrimenti inaccettabile possono essere riportati contattando il team del progetto tramite [INSERIRE INDIRIZZO EMAIL]. Tutti i reclami saranno revisionati ed indagati e risulteranno in una risposta ritenuta necessaria ed appropriata alle circostanze. I team del progetto è obbligato a mantenere riserbo rispetto a chi riporta un caso.
+Ulteriori dettagli di applicazioni specifiche possono essere postati separatamente.
+
+I manutentori del progetto che non seguono o applicano il Codice di Condotta in buona fede possono affrontare ripercussioni temporanee o permanenti in base alla deliberazione di altri membri della direzione del progetto.
```
I **responsabili** del progetto che non seguono o applicano il **Patto di Condotta** in buona fede possono affrontare ripercussioni temporanee o permanenti in base alla deliberazione **degli** altri membri della direzione del progetto.

- manutentori -> responsabili
- Codice di Condotta -> Patto di Condotta
- aggiunto *degli*

amencarini commented on version/1/4/it/code_of_conduct.md:
```diff
@@ -0,0 +1,47 @@
+
+## Applicazione
+
+Casi di comportamento ingiurioso, molesto o altrimenti inaccettabile possono essere riportati contattando il team del progetto tramite [INSERIRE INDIRIZZO EMAIL]. Tutti i reclami saranno revisionati ed indagati e risulteranno in una risposta ritenuta necessaria ed appropriata alle circostanze. I team del progetto è obbligato a mantenere riserbo rispetto a chi riporta un caso.
```
Casi di comportamento ingiurioso, molesto o altrimenti inaccettabile possono essere riportati contattando il team del progetto tramite [INSERIRE INDIRIZZO EMAIL]. Tutti i reclami saranno **esaminati** ed indagati e risulteranno in una risposta ritenuta necessaria ed appropriata alle circostanze. **Il** team del progetto è obbligato a mantenere riserbo rispetto a chi riporta un caso.

- revisionati -> esaminati
- I team -> Il team

amencarini commented on version/1/4/it/code_of_conduct.md:
```diff
@@ -0,0 +1,47 @@
+
+## Ambito
+
+Questo Codice di Comportamento è applicabile sia al progetto online che agli spazi pubblici quando un individuo rappresenta il progetto stesso o la sua comunità. Esempi di rappresentazione di un progetto o una comunità includono l'uso di un indirizzo e-mail ufficiale del progetto, il post in un account ufficiale di un social media, o l'agire come un rappresentante designato in un evento offline o ...
```
Questo **Patto di Condotta** è applicabile sia al **progetto che** agli spazi pubblici quando un individuo rappresenta il progetto stesso o la sua comunità. Esempi di rappresentazione di un progetto o una comunità includono l'uso di un indirizzo e-mail ufficiale del progetto, **post tramite un account ufficiale sui social media**, o l'agire come un rappresentante designato in un evento offline o online. La **rappresentanza** di un progetto può essere ulteriormente **definita e chiarificata dai responsabili** del progetto.

- Codice di Comportamento -> Patto di Condotta (o Codice di Condotta se il primo commento non verrà incorporato)
- progetto online -> progetto
- il post in un account ufficiale di un social media -> post tramite un account ufficiale sui social media
- rappresentazione -> rappresentanza
- definito e chiarificato dai manutentori -> definita e chiarificata dai responsabili

amencarini commented on version/1/4/it/code_of_conduct.md:
```diff
@@ -0,0 +1,47 @@
+
+I mantenitori del progetto sono responsabili nel chiarire gli standard di comportamento accettabile e sono supposti di prendere azioni correttive appropriate e giuste in risposta a qualsiasi caso di comportamento inaccettabile.
+
+I responsabili del progetto hanno il diritto e la responsabilità di rimuovere, modificare, o cancellare commenti, commit, codice, modifiche del wiki, ticket, ed altri contributi che non sono in linea con questo Codice di Comportamento, o di bandire temporaneamente o permanentemente qualsiasi collaboratore per altri comportamenti che verranno ritenuti inappropriati, intimidatori, offensivi o danno ...
```
I responsabili del progetto hanno il diritto e la responsabilità di rimuovere, modificare, o cancellare commenti, commit, codice, modifiche del wiki, ticket, ed altri contributi che non sono in linea con questo Codice di Comportamento, o di bandire temporaneamente o permanentemente qualsiasi collaboratore per altri **comportamenti ritenuti** inappropriati, intimidatori, offensivi o dannosi.

amencarini commented on version/1/4/it/code_of_conduct.md:
```diff
@@ -0,0 +1,47 @@
+# Codice di Condotta Convenuto del Collaboratore
```
**Patto di Condotta dei Collaboratori**

Dasc3er replied:
In questo caso mi ero basato sul titolo della versione precedente, ma credevo anch'io che si potesse migliorare.
Grazie per le altre correzioni, a breve vedo di inserirle! 👍

amencarini reviewed (commented):
Note generali:
- "maintainers" è stato tradotto sia come "manutentori" che "responsabili", ho commentato tutte le istanze di "manutentori" in modo da avere una singola traduzione consistente
- Code of Conduct è stato tradotto in un paio di modi diversi, anche qui ho commentato per consistenza
- I commenti hanno le modifiche proposte in grassetto. Sui paragrafi più lunghi le ho riportate sotto in quanto non erano sempre ben visibili

Dasc3er committed:
Variuos fixes

Variuos fixes suggested by @amencarini

amencarini commented:
👍

nirnaeth commented on version/1/4/it/code_of_conduct.md:
```diff
@@ -0,0 +1,47 @@
+
+Esempi di comportamento inaccettabile da parte dei partecipanti includono:
+
+* L'uso di linguaggio o immagini sessuali e attenzioni o advances sgraditi
```
words from foreign languages don't have the plural `s` in italian

amencarini replied:
Actually, in Italian the French plural version is favoured: "attenzioni o avances sgradite" :)

Dasc3er replied:
Ah, that's it! I was trying to understand what felt wrong about it... :)

nirnaeth replied:
I didn't know about the French thing. TIL :D

nirnaeth commented on version/1/4/it/code_of_conduct.md:
```diff
@@ -0,0 +1,47 @@
+Esempi di comportamento inaccettabile da parte dei partecipanti includono:
+
+* L'uso di linguaggio o immagini sessuali e attenzioni o advances sgraditi
+* Comportamento da troll o commenti offensivi/dispregiativi, e attacchi personali o politici
```
no commas before conjunctions (`, e`)

nirnaeth commented on version/1/4/it/code_of_conduct.md:
```diff
@@ -0,0 +1,47 @@
+
+I responsabili del progetto sono tenuti a chiarire gli standard di comportamento accettabile e a prendere azioni correttive appropriate e giuste in risposta a qualsiasi caso di comportamento inaccettabile.
+
+I responsabili del progetto hanno il diritto e la responsabilità di rimuovere, modificare, o cancellare commenti, commit, codice, modifiche del wiki, ticket, ed altri contributi che non sono in linea con questo Patto di Condotta, o di bandire temporaneamente o permanentemente qualsiasi collaboratore per altri comportamenti ritenuti inappropriati, intimidatori, offensivi o dannosi.
```
no comma in these cases:
* `modificare o cancellare` 
* `ticket ed altri`

Dasc3er replied:
This is a little remaining of 1.3 translation...
It's not really an error, because, if I'm not wrong, Italian is quite flexible in these cases.

We should also talk about this 👍 
@amencarini

nirnaeth commented on version/1/4/it/code_of_conduct.md:
```diff
@@ -0,0 +1,47 @@
+# Patto di Condotta dei Collaboratori
```
I am not really fond of the word `Patto`, I'd stay closer to the original and call it `Codice`, also because the "literature" is already going in that direction (ex. [Italian version of the Berlin Code of Conduct](http://berlincodeofconduct.org/it/))

Dasc3er replied:
Actually, we may translate in two different ways: `Codice di Condotta Convenuto del Collaboratore` or `Patto di Condotta dei Collaboratori`. I would prefer `Patto` because, in one word, we can say  `Codice [...] Convenuto`, which literally translates `Covenant Code`.
The difference between this and the choice done by [Berlin Code of Conduct](http://berlincodeofconduct.org/) is related to the fact that the Berlin Code of Conduct doesn't have `Covenant `on its name.

I believe we should choose together.

nirnaeth replied:
🤔 
I see your point. it makes sense, then, to stick to `Patto`, definitely.

Dasc3er committed:
Comma's fixes

Fixed wrong comma's translations (suggested by @nirnaeth, also with
@amencarini) and some "ed"s.
